BankSDK/GiniBankSDK/Pod/README.md

```diff
@@ -4,12 +4,28 @@ Distributing Gini Bank SDK XCFrameworks using CocoaPods
 How to release new versions
 ---------------------------
 
-1. After the `.xcframeworks` have been created compress them into a zip named `GiniBankSDK-XCFrameworks.zip`.
-2. Update the version in the `GiniBankSDK.podspec`, for example `spec.version = 2.2.1`.
-3. Clone the `https://github.com/gini/gini-podspecs` repository.
-4. Go into the `gini-podspecs` repository folder and create a subfolder in `GiniBankSDK` which is named after the version, for example `GiniBankSDK/2.2.1`.
-5. Copy the `GiniBankSDK-XCFrameworks.zip` and the `GiniBankSDK.podspec` into the new folder.
-6. Commit and push the new folder and its contents.
+1. After the `.xcframeworks` have been created compress them into a zip named `GiniBankSDK-XCFrameworks.zip`.  
+   IMPORTANT: On MacOS the zip needs to be created by selecting all the XCFrameworks and compressing them. It won't work
+   if you compress the folder containing the XCFrameworks[^1].  
+   You can also use the following `zip` command inside the folder where all the XCFrameworks are (make sure you include
+   all the required XCFrameworks, this example might be outdated):
+   ```
+   zip -r GiniBankSDK-XCFrameworks.zip \
+   GiniBankAPILibrary.xcframework \
+   GiniBankSDK.xcframework \
+   GiniCaptureSDK.xcframework
+   ```
+1. Update the version in the `GiniBankSDK.podspec`, for example `spec.version = 2.2.1`.
+2. Clone the `https://github.com/gini/gini-podspecs` repository.
+3. Go into the `gini-podspecs` repository folder and create a subfolder in `GiniBankSDK` which is named after the
+   version, for example `GiniBankSDK/2.2.1`.
+4. Copy the `GiniBankSDK-XCFrameworks.zip` and the updated `GiniBankSDK.podspec` into the new folder.
+5. Commit and push the new folder and its contents to `gini-podspecs`. You can discard the changes in the
+   `gini-mobile-ios` repo.
+
+[^1]: If we put the XCFrameworks into a folder and then zip that folder then the `unzip` command Cocoapods uses will put
```

Comment:
[Clarification]: could you please add the exact `unzip` command, that you use for testing in the terminal? 🥹